### ISSUE TITLE
[Feature]: Enhancing social handles accessibility in Footer

### ISSUE THREAD

**Issue by Avinash905** [sourabhsikarwar/Scene-Movie-Platform#405]
- [x] Would you like to take up this task? 
- [x] Are you from GSSoC'23? 

|   ✨  |   Feature request                |
|-----|---------------------------------|
| **Is your feature request related to a problem?** | Currently, the problem lies in the absence of the aria attributes for these social handles. |
| **Describe the solution you'd like** | By implementing these ARIA attributes, we can significantly enhance the accessibility ensuring all users can effectively interact with our application also would add **rel** for the anchors which are lacking to ensure the best practices for crawlers |

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.
